modules/@apostrophecms/i18n/i18n/sk.json

```diff
@@ -131,7 +131,7 @@
   "fileTypeNotAccepted": "Typ súboru nebol prijatý. Povolené rozšírenia: {{ extensions }}",
   "fileInvalid": "Súbor nebol prijatý. Môže byť poškodený alebo sa jeho obsah nemusí zhodovať s príponou súboru.",
   "files": "Súbory",
-  "filter": "Filter",
+  "filter": "filtrovať",
```

Comment:
I'm sorry to do this again, but I think this should be capitalized. `Filtrovať`, correct?

Reply:
You have an eye for detail. Very cool. 

Fix:  b1adccf8123994e1aa5cced3522aa1967b6e54f0

Reply:
I'm testing the translation locally now. I'd like to change another word. Type should be "Typ". 

Fix: b2b84460e912baeec5016d86e9f935b8ef580133

Reply:
I couple more changes to credit and creditURl.

Fix: aa17cd01ff4724b94611d0fbfbe85b8f595fefbf